protolite-well-known-types/protolite-well-known-types.gradle

```diff
@@ -50,13 +50,9 @@ android {
     sourceSets.main.resources.includes = ['**/*.proto']
 }
 
-configurations.all {
-    exclude group: "com.google.protobuf", module: "protobuf-java"
-}
-
 dependencies {
     implementation 'com.google.protobuf:protobuf-javalite:3.11.0'
-    protobuf 'com.google.protobuf:protobuf-javalite:3.11.0'
+    protobuf 'com.google.protobuf:protobuf-java:3.11.4'
```

Comment:
The `protobuf` declaration is different from a `implementation`.

IIRC, this will simply make the proto files in the artifact available to all consumers of this dep. It should not vendor the protos itself. It should simply make it available to the proto compiler